src/sentry/static/sentry/app/stores/guideStore.tsx

```diff
@@ -1,32 +1,58 @@
 import {browserHistory} from 'react-router';
 import Reflux from 'reflux';
-import GuideActions from 'app/actions/guideActions';
-import OrganizationsActions from 'app/actions/organizationsActions';
-import ProjectActions from 'app/actions/projectActions';
+import * as Sentry from '@sentry/browser';
+
 import {Client} from 'app/api';
-import ConfigStore from 'app/stores/configStore';
+import {Guide, Guides, GuideStep, GuideData} from 'app/components/assistant/types';
+import {LightWeightOrganization} from 'app/types';
 import {trackAnalyticsEvent} from 'app/utils/analytics';
+import ConfigStore from 'app/stores/configStore';
+import getGuideContent from 'app/components/assistant/getGuideContent';
+import GuideActions from 'app/actions/guideActions';
+import OrganizationsActions from 'app/actions/organizationsActions';
+
+type State = {
+  guides: Guides;
+  anchors: Set<string>;
+  currentGuide: Guide | null;
+  currentStep: number;
+  orgId: string | null;
+  forceShow: boolean;
+  prevGuide: Guide | null;
+};
 
 const GuideStore = Reflux.createStore({
   init() {
     this.state = {
-      // All guides returned to us from the server.
+      /**
+       * All tooltip guides
+       */
       guides: {},
-      // All anchors that are currently mounted.
+      /**
+       * Anchors that are currently mounted
+       */
       anchors: new Set(),
-      // The "on deck" guide.
+      /**
+       * The current guide
+       */
       currentGuide: null,
-      // Current step of the current guide (0-indexed).
+      /**
+       * Current step of the current guide
+       */
       currentStep: 0,
-      // Current organization.
-      org: null,
-      // Current project.
-      project: null,
-      // We force show a guide if the URL contains #assistant.
+      /**
+       * Current organization id
+       */
+      orgId: null,
+      /**
+       * We force show a guide if the URL contains #assistant
+       */
       forceShow: false,
-      // The previously shown guide.
+      /**
+       * The previously shown guide
+       */
       prevGuide: null,
-    };
+    } as State;
```

Comment:
Might be worthwhile to see how I did some of this in the hookStore.tsx file. I think there's some gotchas